src/main/java/Counter.java

```diff
@@ -1,84 +1,43 @@
-import java.util.HashSet;
-import java.util.Set;
-
 /**
  * Counter
  */
-public class Counter implements CRDT {
+public class Counter extends CRDT {
     private static final long serialVersionUID = 1L;
-    private Set<Integer> IdSet;
-
     int count;
 
+    public Class<?>[] increment = new Class[] { Integer.class };
+    public Class<?>[] decrement = new Class[] { Integer.class };
```

Comment:
interesting.  What's the pattern when there are multiple args?

Reply:
See GrowthMap.java. It's not actually a CRDT(Order of operations matters) so it will be worth checking to see what is the convenient solution to make it convergent.